core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -714,6 +714,24 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  /**
+   * Upload a file to a Hadoop-compatible filesystem.
+   *
+   */
+  def uploadHcfsFile(
```

Comment:
I don't think we need a whole global utility method for this. It's used one place and is almost a one-liner. I also don't think I've seen "HCFS"? it sounds like a new file system but it isn't

Reply:
fwiw, I had never seen "HCFS" either

Reply:
Note the second link, the notion of Hadoop compatibility is covered in current Hadoop documentation for all maintained branches:

- 3.1.x - https://hadoop.apache.org/docs/r3.1.1/hadoop-project-dist/hadoop-common/filesystem/introduction.html#Core_Expectations_of_a_Hadoop_Compatible_FileSystem
- 2.9.x - https://hadoop.apache.org/docs/r2.9.2/hadoop-project-dist/hadoop-common/filesystem/introduction.html#Core_Expectations_of_a_Hadoop_Compatible_FileSystem
- 2.8.x - https://hadoop.apache.org/docs/r2.8.5/hadoop-project-dist/hadoop-common/filesystem/introduction.html
- 2.7.x - https://hadoop.apache.org/docs/r2.7.7/hadoop-project-dist/hadoop-common/filesystem/introduction.html#Core_Expectations_of_a_Hadoop_Compatible_FileSystem

Reply:
To be clear I'm not disputing the idea of "Hadoop Compatible File System", but the acronym HCFS. It sounds like a particular file system, when it's just a descriptive category. I'd write it out.

Reply:
I can change it... I will try think a better name,